{"nodes":[{"pos":[801,904],"content":"The exception that is thrown when a segment directory is missing from the pipeline directory structure.","needQuote":true,"needEscape":true,"nodes":[{"content":"The exception that is thrown when a segment directory is missing from the pipeline directory structure.","pos":[0,103]}]},{"pos":[916,1235],"content":"When an add-in is activated, its pipeline to the host is constructed. The add-in system requires that pipeline segments be placed in a specified directory structure with specified directory names, as described in [Pipeline Development Requirements](http://msdn.microsoft.com/en-us/ef9fa986-e80b-43e1-868b-247f4c1d9da5).","needQuote":true,"needEscape":true,"nodes":[{"content":"When an add-in is activated, its pipeline to the host is constructed.","pos":[0,69]},{"content":"The add-in system requires that pipeline segments be placed in a specified directory structure with specified directory names, as described in <bpt id=\"p1\">[</bpt>Pipeline Development Requirements<ept id=\"p1\">](http://msdn.microsoft.com/en-us/ef9fa986-e80b-43e1-868b-247f4c1d9da5)</ept>.","pos":[70,319],"source":" The add-in system requires that pipeline segments be placed in a specified directory structure with specified directory names, as described in [Pipeline Development Requirements](http://msdn.microsoft.com/en-us/ef9fa986-e80b-43e1-868b-247f4c1d9da5)."}]},{"pos":[1955,2076],"content":"Initializes a new instance of the <bpt id=\"p1\">&lt;xref href=\"System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException\"&gt;</bpt><ept id=\"p1\">&lt;/xref&gt;</ept> class.","needQuote":true,"needEscape":true,"source":"Initializes a new instance of the <xref href=\"System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException\"></xref> class."},{"pos":[2089,2752],"content":"This constructor initializes the <xref:System.Exception.Message%2A> property of the new instance to a system-supplied message that describes the error (for example, \"The target application domain has been unloaded.\"). This message takes the current system culture into account.       The following table shows the initial property values for an instance of the <xref:System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException> class.      |Property|Value|   |--------------|-----------|   |<xref:System.Exception.InnerException%2A>|A null reference (`Nothing` in Visual Basic).|   |<xref:System.Exception.Message%2A>|The localized error message string.|","needQuote":false,"needEscape":true,"nodes":[{"content":"This constructor initializes the &lt;xref:System.Exception.Message%2A&gt; property of the new instance to a system-supplied message that describes the error (for example, \"The target application domain has been unloaded.\").","pos":[0,217],"source":"This constructor initializes the <xref:System.Exception.Message%2A> property of the new instance to a system-supplied message that describes the error (for example, \"The target application domain has been unloaded.\")."},{"content":"This message takes the current system culture into account.","pos":[218,277]},{"content":"The following table shows the initial property values for an instance of the &lt;xref:System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException&gt; class.","pos":[284,434],"source":"       The following table shows the initial property values for an instance of the <xref:System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException> class."},{"content":"|Property|Value|   |--------------|-----------|   |&lt;xref:System.Exception.InnerException%2A&gt;|A null reference (<ph id=\"ph1\">`Nothing`</ph> in Visual Basic).|   |&lt;xref:System.Exception.Message%2A&gt;|The localized error message string.|","pos":[440,654],"source":"      |Property|Value|   |--------------|-----------|   |<xref:System.Exception.InnerException%2A>|A null reference (`Nothing` in Visual Basic).|   |<xref:System.Exception.Message%2A>|The localized error message string.|"}]},{"pos":[3544,3690],"content":"Initializes a new instance of the <bpt id=\"p1\">&lt;xref href=\"System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException\"&gt;</bpt><ept id=\"p1\">&lt;/xref&gt;</ept> class with a specified message.","needQuote":true,"needEscape":true,"source":"Initializes a new instance of the <xref href=\"System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException\"></xref> class with a specified message."},{"pos":[3703,4078],"content":"The following table shows the initial property values for an instance of the <xref:System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException> class.      |Property|Value|   |--------------|-----------|   |<xref:System.Exception.InnerException%2A>|A null reference (`Nothing` in Visual Basic).|   |<xref:System.Exception.Message%2A>|The localized error message string.|","needQuote":false,"needEscape":true,"nodes":[{"content":"The following table shows the initial property values for an instance of the &lt;xref:System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException&gt; class.","pos":[0,150],"source":"The following table shows the initial property values for an instance of the <xref:System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException> class."},{"content":"|Property|Value|   |--------------|-----------|   |&lt;xref:System.Exception.InnerException%2A&gt;|A null reference (<ph id=\"ph1\">`Nothing`</ph> in Visual Basic).|   |&lt;xref:System.Exception.Message%2A&gt;|The localized error message string.|","pos":[156,370],"source":"      |Property|Value|   |--------------|-----------|   |<xref:System.Exception.InnerException%2A>|A null reference (`Nothing` in Visual Basic).|   |<xref:System.Exception.Message%2A>|The localized error message string.|"}]},{"pos":[4246,4281],"content":"A message that describes the error.","needQuote":true,"needEscape":true,"nodes":[{"content":"A message that describes the error.","pos":[0,35]}]},{"pos":[5222,5367],"content":"Initializes a new instance of the <bpt id=\"p1\">&lt;xref href=\"System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException\"&gt;</bpt><ept id=\"p1\">&lt;/xref&gt;</ept> class with serialization data.","needQuote":true,"needEscape":true,"source":"Initializes a new instance of the <xref href=\"System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException\"></xref> class with serialization data."},{"pos":[5380,5979],"content":"This constructor is called during deserialization to reconstitute an exception object that is transmitted over a stream. For more information, see [XML and SOAP Serialization](~/add/includes/ajax-current-ext-md.md).       The following table shows the initial property values for an instance of the <xref:System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException> class.      |Property|Value|   |--------------|-----------|   |<xref:System.Exception.InnerException%2A>|A null reference (`Nothing` in Visual Basic).|   |<xref:System.Exception.Message%2A>|The localized error message string.|","needQuote":false,"needEscape":true,"nodes":[{"content":"This constructor is called during deserialization to reconstitute an exception object that is transmitted over a stream.","pos":[0,120]},{"content":"For more information, see <bpt id=\"p1\">[</bpt>XML and SOAP Serialization<ept id=\"p1\">](~/add/includes/ajax-current-ext-md.md)</ept>.","pos":[121,215],"source":" For more information, see [XML and SOAP Serialization](~/add/includes/ajax-current-ext-md.md)."},{"content":"The following table shows the initial property values for an instance of the &lt;xref:System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException&gt; class.","pos":[222,372],"source":"       The following table shows the initial property values for an instance of the <xref:System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException> class."},{"content":"|Property|Value|   |--------------|-----------|   |&lt;xref:System.Exception.InnerException%2A&gt;|A null reference (<ph id=\"ph1\">`Nothing`</ph> in Visual Basic).|   |&lt;xref:System.Exception.Message%2A&gt;|The localized error message string.|","pos":[378,592],"source":"      |Property|Value|   |--------------|-----------|   |<xref:System.Exception.InnerException%2A>|A null reference (`Nothing` in Visual Basic).|   |<xref:System.Exception.Message%2A>|The localized error message string.|"}]},{"pos":[6272,6321],"content":"The object that holds the serialized object data.","needQuote":true,"needEscape":true,"nodes":[{"content":"The object that holds the serialized object data.","pos":[0,49]}]},{"pos":[6417,6488],"content":"The contextual information about the source or destination object data.","needQuote":true,"needEscape":true,"nodes":[{"content":"The contextual information about the source or destination object data.","pos":[0,71]}]},{"pos":[7251,7478],"content":"Initializes a new instance of the <bpt id=\"p1\">&lt;xref href=\"System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException\"&gt;</bpt><ept id=\"p1\">&lt;/xref&gt;</ept> class with a specified error message and a reference to the inner exception that is the cause of this exception.","needQuote":true,"needEscape":true,"source":"Initializes a new instance of the <xref href=\"System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException\"></xref> class with a specified error message and a reference to the inner exception that is the cause of this exception."},{"pos":[7491,8295],"content":"An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the <xref:System.Exception.InnerException%2A> property. The <xref:System.Exception.InnerException%2A> property returns the same value that is passed to the constructor, or a null reference if the <xref:System.Exception.InnerException%2A> property does not supply the inner exception value to the constructor.       The following table shows the initial property values for an instance of the <xref:System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException> class.      |Property|Value|   |--------------|-----------|   |<xref:System.Exception.InnerException%2A>|The inner exception reference.|   |<xref:System.Exception.Message%2A>|The localized error message string.|","needQuote":false,"needEscape":true,"nodes":[{"content":"An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the <xref:System.Exception.InnerException%2A> property. The <xref:System.Exception.InnerException%2A> property returns the same value that is passed to the constructor, or a null reference if the <xref:System.Exception.InnerException%2A> property does not supply the inner exception value to the constructor.       The following table shows the initial property values for an instance of the <xref:System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException> class.      |Property|Value|   |--------------|-----------|   |<xref:System.Exception.InnerException%2A>|The inner exception reference.|   |<xref:System.Exception.Message%2A>|The localized error message string.|","pos":[0,797],"nodes":[{"content":"An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the &lt;xref:System.Exception.InnerException%2A&gt; property.","pos":[0,182],"source":"An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the <xref:System.Exception.InnerException%2A> property."},{"content":"The &lt;xref:System.Exception.InnerException%2A&gt; property returns the same value that is passed to the constructor, or a null reference if the &lt;xref:System.Exception.InnerException%2A&gt; property does not supply the inner exception value to the constructor.","pos":[183,435],"source":" The <xref:System.Exception.InnerException%2A> property returns the same value that is passed to the constructor, or a null reference if the <xref:System.Exception.InnerException%2A> property does not supply the inner exception value to the constructor."},{"content":"The following table shows the initial property values for an instance of the &lt;xref:System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException&gt; class.","pos":[442,592],"source":"       The following table shows the initial property values for an instance of the <xref:System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException> class."},{"content":"|Property|Value|   |--------------|-----------|   |&lt;xref:System.Exception.InnerException%2A&gt;|The inner exception reference.|   |&lt;xref:System.Exception.Message%2A&gt;|The localized error message string.|","pos":[598,797],"source":"      |Property|Value|   |--------------|-----------|   |<xref:System.Exception.InnerException%2A>|The inner exception reference.|   |<xref:System.Exception.Message%2A>|The localized error message string.|"}]}]},{"pos":[8489,8524],"content":"A message that describes the error.","needQuote":true,"needEscape":true,"nodes":[{"content":"A message that describes the error.","pos":[0,35]}]},{"pos":[8598,8797],"content":"The exception that is the cause of the current exception. If the `innerException` parameter is not a null reference, the current exception is raised in a catch block that handles the inner exception.","needQuote":true,"needEscape":true,"nodes":[{"content":"The exception that is the cause of the current exception.","pos":[0,57]},{"content":"If the <ph id=\"ph1\">`innerException`</ph> parameter is not a null reference, the current exception is raised in a catch block that handles the inner exception.","pos":[58,199],"source":" If the `innerException` parameter is not a null reference, the current exception is raised in a catch block that handles the inner exception."}]}],"content":"### YamlMime:ManagedReference\nitems:\n- uid: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException\n  id: AddInSegmentDirectoryNotFoundException\n  children:\n  - System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException.#ctor\n  - System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException.#ctor(System.String)\n  - System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)\n  - System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException.#ctor(System.String,System.Exception)\n  langs:\n  - csharp\n  name: AddInSegmentDirectoryNotFoundException\n  nameWithType: AddInSegmentDirectoryNotFoundException\n  fullName: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException\n  type: Class\n  summary: The exception that is thrown when a segment directory is missing from the pipeline directory structure.\n  remarks: When an add-in is activated, its pipeline to the host is constructed. The add-in system requires that pipeline segments be placed in a specified directory structure with specified directory names, as described in [Pipeline Development Requirements](http://msdn.microsoft.com/en-us/ef9fa986-e80b-43e1-868b-247f4c1d9da5).\n  syntax:\n    content: 'public class AddInSegmentDirectoryNotFoundException : Exception'\n  inheritance:\n  - System.Exception\n  implements: []\n  inheritedMembers: []\n  platform:\n  - net462\n- uid: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException.#ctor\n  id: '#ctor'\n  parent: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException\n  langs:\n  - csharp\n  name: AddInSegmentDirectoryNotFoundException()\n  nameWithType: AddInSegmentDirectoryNotFoundException.AddInSegmentDirectoryNotFoundException()\n  fullName: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException.AddInSegmentDirectoryNotFoundException()\n  type: Constructor\n  assemblies:\n  - System.AddIn\n  namespace: System.AddIn.Hosting\n  summary: Initializes a new instance of the <xref href=\"System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException\"></xref> class.\n  remarks: \"This constructor initializes the <xref:System.Exception.Message%2A> property of the new instance to a system-supplied message that describes the error (for example, \\\"The target application domain has been unloaded.\\\"). This message takes the current system culture into account.  \\n  \\n The following table shows the initial property values for an instance of the <xref:System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException> class.  \\n  \\n|Property|Value|  \\n|--------------|-----------|  \\n|<xref:System.Exception.InnerException%2A>|A null reference (`Nothing` in Visual Basic).|  \\n|<xref:System.Exception.Message%2A>|The localized error message string.|\"\n  syntax:\n    content: public AddInSegmentDirectoryNotFoundException ();\n    parameters: []\n  overload: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException.#ctor*\n  exceptions: []\n  platform:\n  - net462\n- uid: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException.#ctor(System.String)\n  id: '#ctor(System.String)'\n  parent: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException\n  langs:\n  - csharp\n  name: AddInSegmentDirectoryNotFoundException(String)\n  nameWithType: AddInSegmentDirectoryNotFoundException.AddInSegmentDirectoryNotFoundException(String)\n  fullName: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException.AddInSegmentDirectoryNotFoundException(String)\n  type: Constructor\n  assemblies:\n  - System.AddIn\n  namespace: System.AddIn.Hosting\n  summary: Initializes a new instance of the <xref href=\"System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException\"></xref> class with a specified message.\n  remarks: \"The following table shows the initial property values for an instance of the <xref:System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException> class.  \\n  \\n|Property|Value|  \\n|--------------|-----------|  \\n|<xref:System.Exception.InnerException%2A>|A null reference (`Nothing` in Visual Basic).|  \\n|<xref:System.Exception.Message%2A>|The localized error message string.|\"\n  syntax:\n    content: public AddInSegmentDirectoryNotFoundException (string message);\n    parameters:\n    - id: message\n      type: System.String\n      description: A message that describes the error.\n  overload: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException.#ctor*\n  exceptions: []\n  platform:\n  - net462\n- uid: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)\n  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'\n  parent: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException\n  langs:\n  - csharp\n  name: AddInSegmentDirectoryNotFoundException(SerializationInfo,StreamingContext)\n  nameWithType: AddInSegmentDirectoryNotFoundException.AddInSegmentDirectoryNotFoundException(SerializationInfo,StreamingContext)\n  fullName: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException.AddInSegmentDirectoryNotFoundException(SerializationInfo,StreamingContext)\n  type: Constructor\n  assemblies:\n  - System.AddIn\n  namespace: System.AddIn.Hosting\n  summary: Initializes a new instance of the <xref href=\"System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException\"></xref> class with serialization data.\n  remarks: \"This constructor is called during deserialization to reconstitute an exception object that is transmitted over a stream. For more information, see [XML and SOAP Serialization](~/add/includes/ajax-current-ext-md.md).  \\n  \\n The following table shows the initial property values for an instance of the <xref:System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException> class.  \\n  \\n|Property|Value|  \\n|--------------|-----------|  \\n|<xref:System.Exception.InnerException%2A>|A null reference (`Nothing` in Visual Basic).|  \\n|<xref:System.Exception.Message%2A>|The localized error message string.|\"\n  syntax:\n    content: protected AddInSegmentDirectoryNotFoundException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);\n    parameters:\n    - id: info\n      type: System.Runtime.Serialization.SerializationInfo\n      description: The object that holds the serialized object data.\n    - id: context\n      type: System.Runtime.Serialization.StreamingContext\n      description: The contextual information about the source or destination object data.\n  overload: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException.#ctor*\n  exceptions: []\n  platform:\n  - net462\n- uid: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException.#ctor(System.String,System.Exception)\n  id: '#ctor(System.String,System.Exception)'\n  parent: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException\n  langs:\n  - csharp\n  name: AddInSegmentDirectoryNotFoundException(String,Exception)\n  nameWithType: AddInSegmentDirectoryNotFoundException.AddInSegmentDirectoryNotFoundException(String,Exception)\n  fullName: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException.AddInSegmentDirectoryNotFoundException(String,Exception)\n  type: Constructor\n  assemblies:\n  - System.AddIn\n  namespace: System.AddIn.Hosting\n  summary: Initializes a new instance of the <xref href=\"System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException\"></xref> class with a specified error message and a reference to the inner exception that is the cause of this exception.\n  remarks: \"An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the <xref:System.Exception.InnerException%2A> property. The <xref:System.Exception.InnerException%2A> property returns the same value that is passed to the constructor, or a null reference if the <xref:System.Exception.InnerException%2A> property does not supply the inner exception value to the constructor.  \\n  \\n The following table shows the initial property values for an instance of the <xref:System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException> class.  \\n  \\n|Property|Value|  \\n|--------------|-----------|  \\n|<xref:System.Exception.InnerException%2A>|The inner exception reference.|  \\n|<xref:System.Exception.Message%2A>|The localized error message string.|\"\n  syntax:\n    content: public AddInSegmentDirectoryNotFoundException (string message, Exception innerException);\n    parameters:\n    - id: message\n      type: System.String\n      description: A message that describes the error.\n    - id: innerException\n      type: System.Exception\n      description: The exception that is the cause of the current exception. If the `innerException` parameter is not a null reference, the current exception is raised in a catch block that handles the inner exception.\n  overload: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException.#ctor*\n  exceptions: []\n  platform:\n  - net462\nreferences:\n- uid: System.Exception\n  isExternal: false\n  name: System.Exception\n- uid: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException.#ctor\n  parent: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException\n  isExternal: false\n  name: AddInSegmentDirectoryNotFoundException()\n  nameWithType: AddInSegmentDirectoryNotFoundException.AddInSegmentDirectoryNotFoundException()\n  fullName: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException.AddInSegmentDirectoryNotFoundException()\n- uid: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException.#ctor(System.String)\n  parent: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException\n  isExternal: false\n  name: AddInSegmentDirectoryNotFoundException(String)\n  nameWithType: AddInSegmentDirectoryNotFoundException.AddInSegmentDirectoryNotFoundException(String)\n  fullName: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException.AddInSegmentDirectoryNotFoundException(String)\n- uid: System.String\n  parent: System\n  isExternal: true\n  name: String\n  nameWithType: String\n  fullName: System.String\n- uid: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)\n  parent: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException\n  isExternal: false\n  name: AddInSegmentDirectoryNotFoundException(SerializationInfo,StreamingContext)\n  nameWithType: AddInSegmentDirectoryNotFoundException.AddInSegmentDirectoryNotFoundException(SerializationInfo,StreamingContext)\n  fullName: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException.AddInSegmentDirectoryNotFoundException(SerializationInfo,StreamingContext)\n- uid: System.Runtime.Serialization.SerializationInfo\n  parent: System.Runtime.Serialization\n  isExternal: false\n  name: SerializationInfo\n  nameWithType: SerializationInfo\n  fullName: System.Runtime.Serialization.SerializationInfo\n- uid: System.Runtime.Serialization.StreamingContext\n  parent: System.Runtime.Serialization\n  isExternal: true\n  name: StreamingContext\n  nameWithType: StreamingContext\n  fullName: System.Runtime.Serialization.StreamingContext\n- uid: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException.#ctor(System.String,System.Exception)\n  parent: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException\n  isExternal: false\n  name: AddInSegmentDirectoryNotFoundException(String,Exception)\n  nameWithType: AddInSegmentDirectoryNotFoundException.AddInSegmentDirectoryNotFoundException(String,Exception)\n  fullName: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException.AddInSegmentDirectoryNotFoundException(String,Exception)\n- uid: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException.#ctor*\n  parent: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException\n  isExternal: false\n  name: AddInSegmentDirectoryNotFoundException\n  nameWithType: AddInSegmentDirectoryNotFoundException.AddInSegmentDirectoryNotFoundException\n"}